app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-input.tsx

```diff
@@ -1,80 +0,0 @@
-import { forwardRef, useCallback } from 'react';
-import type { ForwardedRef } from 'react';
-import Cleave from 'cleave.js/react';
-import type { ReactInstanceWithCleave } from 'cleave.js/react/props';
```

Comment:
This reminds me there are some dependencies that I think we can now remove from the package's `package.json`.

Specifically these two:

https://github.com/18F/identity-idp/blob/9705e38301ee9469ebd54c90c9a4cc929ed074c4/app/javascript/packages/verify-flow/package.json#L6-L7

Reply:
let me give this a shot